guides/md/keras_cv/custom_image_augmentations.md

```diff
@@ -120,9 +121,9 @@ imshow(elephants)
 ```
 
 
-    
-![png](/img/guides/custom_image_augmentations/custom_image_augmentations_9_0.png)
-    
+
+![png](../guides/img/custom_image_augmentations/custom_image_augmentations_9_0.png)
```

Comment:
The expected image path is `/img/guides/`, not `../guides/img/`. What is causing this?

Reply:
I have no idea.  I didn't change the scripts at all.  I went ahead and update them.